lib/view_component/engine.rb

```diff
@@ -10,6 +10,7 @@ class Engine < Rails::Engine # :nodoc:
     initializer "view_component.set_configs" do |app|
       options = app.config.view_component
 
+      options.render_monkey_patch_enabled = true if options.render_monkey_patch_enabled.nil?
```

Comment:
What if `render_monkey_patch_enabled` is set to true?

Reply:
The default should be `true`. If it is set to `true` in will include the monkey patches. If set to `false` `render_component` gets included in `ActionView` and `ActionController`.